src/com/itacademy/eshop/simulations/CustomerSimulation.java

```diff
@@ -29,21 +36,48 @@ private void addProductsToShoppingCart() {
          * User browses the shop and adds 4 products to the shopping cart.
          * One of them should be a book and other should be a shirt.
          */
-        throw new UnsupportedOperationException("addProductsToShoppingCart() is not implemented yet");
+
+        Product laptop = shop.findProductByName("Cooking Book");
```

Comment:
could be inlined, so you dont need to create 4 separate variables that are only used once